functionaltests/src/test/java/specs/response/HealthResponseFixture.java

```diff
@@ -0,0 +1,36 @@
+package specs.response;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.concordion.api.MultiValueResult;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.json.JSONObject;
+import org.junit.runner.RunWith;
+
+/**
+ * Simple /health URL response checker
+ */
+@RunWith(ConcordionRunner.class)
+public class HealthResponseFixture extends BasePaymentFixture {
+
+    public String getUrl() throws Exception {
+        return getHealthUrl();
+    }
+
+    public MultiValueResult handleHealthResponse() throws Exception {
+        final HttpResponse httpResponse = Request.Get(getHealthUrl())
+                .connectTimeout(200)
```

Comment:
I am still not sure if it is good idea as you always have some network latency which you do not take into account here. By the nature of LB the node which times out will be removed from LB cluster. I would like to omit having payment service down just because of latency issues. @svenmueller is described LB behaviour correct or you have different opinion on that?